src/resources/postcss/customizer-views-v2-controls.pcss

```
@@ -26,5 +26,24 @@
 		.customize-control .customize-inside-control-row {
 			padding: 3px 0;
 		}
+
+		li.customize-control[id$="_choice"] {
+			background-color: pink;
+		}
+
+		&[id$="_choice"] + [id^="customize-control-tribe_customizer"][id$="_color"] {
+			margin-top: -18px;
+		}
+
+		.button.wp-color-result {
+			border-color: #141827;
+
+			&:active,
+			&:focus,
+			&:hover {
+				border-color: #141827;
+				box-shadow: 0px 0px 0px .75px rgba(20, 24, 39, .5);
```

```suggestion
				box-shadow: 0 0 0 0.75px rgba(20, 24, 39, 0.5);
```

though, I think there is already a color for the above, maybe a different alpha value? maybe we can add it to the variables?

---

There is - at least one close enough